include/nbl/builtin/glsl/utils/swapchain_transform.glsl

```diff
@@ -0,0 +1,146 @@
+#ifndef _NBL_BUILTIN_GLSL_SWAPCHAIN_TRANSFORM_INCLUDED_
+#define _NBL_BUILTIN_GLSL_SWAPCHAIN_TRANSFORM_INCLUDED_
+
+#include "nbl/builtin/glsl/utils/swapchain_transform_e.h"
+
+// TODO test transforms outside of horizontal mirror rotate 180 (flip y)
+
+ivec2 nbl_glsl_swapchain_transform_forward(in uint swapchainTransform, in ivec2 coord, in ivec2 lastTexel) {
```

Comment:
fold this function into `postTransform` and set `ivec2 lastTexel = screenSize - ivec2(1)`; as a local var